### DISCUSSION TITLE
Image Size does not work properly

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#8745] [Help] [answered]
Hi there,

I've set an image to be a certain size, which show fine in design mode, but keeps expanding in preview and published version.

Any idea what I might be doing wrong?

App version 2.1.26 with Chrome Version 107.0.5304.110 (Official Build) (arm64)
![Screenshot 2022-11-21 at 12 19 33](https://user-images.githubusercontent.com/4005405/203038152-692b975b-9d04-44c1-aaff-dd0b493c3c64.png)
![Screenshot 2022-11-21 at 12 19 44](https://user-images.githubusercontent.com/4005405/203038172-c160f440-7b29-41e5-9120-9ac53237a5ba.png)

**Comment by melohagan** [comment, marked as answer]
Hey @ecxs 

All you need to do is add a unit to the sizes, so in this case: `25px`.

It is a minor bug though that the builder automatically assumes pixels, but the preview does not. I've raised an issue here: https://github.com/Budibase/budibase/issues/8750

**Reply by ecxs** [reply]
Thanks!
